libretroshare/src/grouter/groutertypes.h

```diff
@@ -21,17 +21,22 @@
  *******************************************************************************/
 #pragma once
 
-#include <stdint.h>
-#include "util/rstime.h"
+#include <cstdint>
 #include <list>
+
+#include "util/rstime.h"
 #include "pgp/rscertificate.h"
 #include "turtle/p3turtle.h"
 #include "retroshare/rsgrouter.h"
 
-class RsGRouterGenericDataItem ;
+struct RsGRouterGenericDataItem;
 class RsGRouterSignedReceiptItem ;
 
-static const uint16_t GROUTER_CLIENT_ID_MESSAGES     = 0x1001 ;
+#ifndef V07_NON_BACKWARD_COMPATIBLE_CHANGE_GROUTER_SERVICE_ID
+/// Pay special attention fixing this, as may break retrocompatibility!
+RS_DEPRECATED_FOR(RsServiceType::MSG)
```

Comment:
MSG? Probably a copy/paste error?

Reply:
this doesn't mean that `RsServiceType::MSG` is deprecated, but that the near defined symbol `GROUTER_CLIENT_ID_MESSAGES` is deprecated and that `RsServiceType::MSG` should be used instead of that

Reply:
That's an error. See my other comments about GRouterId type.